tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/variabledef/TupleVariableDefinitionTest.java

```diff
@@ -439,7 +439,21 @@ public void testNegativeTupleVariables() {
                 "but found 'Ints'", 128, 5);
         BAssertUtil.validateError(resultNegative, ++i, "invalid list binding pattern: expected an array or a tuple, " +
                 "but found 'IntsOrStrings'", 129, 5);
-
+        BAssertUtil.validateError(resultNegative, ++i, "invalid list binding pattern; " +
```

Comment:
Since syntax errors have a lot of diagnostics better not to mix them with negative semantic tests. (to mitigate readability and maintainability issues)
Shall we add this as a separate test. Say `testTupleVarDefnSyntaxNegative()`?

Reply:
Updated. cdc2ba9492ac40f5768731f83dd248276af7bf08